Drivers/Inc/CANbus.h

```diff
@@ -18,59 +20,57 @@ typedef enum {
 	CAR_STATE = 0x580,
 	CHARGE_ENABLE = 0x10C,
 	SUPPLEMENTAL_VOLTAGE = 0x10B,
-	STATE_OF_CHARGE = 0x106
+	STATE_OF_CHARGE = 0x106,
+	MOTOR_DRIVE = 0x221,
+	MOTOR_POWER = 0x222,
+	MOTOR_RESET = 0x223,
+	MOTOR_STATUS = 0x241,
+	MOTOR_VELOCITY = 0x243,
+	DO_NOT_USE_FORCE_ENUM_SIZE = 0xFFFFFFFF //force the enum values to be 32 bits wide. This value should not be used. This is here because the BSP layer expects a 32 bit ptr for ID's
```

Comment:
I think a better solution here would be to add ```-fno-short-enums``` to the Makefile and to use ```__attribute__((packed))``` whenever a short/char sized enum is desired